plugins/file/plugin.go

```diff
@@ -21,16 +26,75 @@ func init() {
 	plugins.RegisterPlugin("file", setupFile6, setupFile4)
 }
 
+//Record holds an IP lease record
+type Record struct {
+	IP        net.IP
```

Comment:
Can't this be simplified to contain only the timestamp of when the lease expires instead of when the lease was assigned plus its duration?